styles.css

```diff
@@ -113,9 +113,52 @@ button {
     width: auto;
 }
 
-@media screen and (max-width: 300px) {
+@media screen and (max-width: 600px) {
     body{
-        background-size: cover;
-        background-color: tan;
+        /* background-size: cover; */
+        background: #134d71;
+    }
+    h1{
+        font-size: 2rem;
+    }
+    .subtitle{
+        margin-top: 5%;
+    }
+    .affirmation, .mantra {
+        margin: 1em 0em 1em 1em;
+        font-size: 1em;
+        transform: scale(1);
+        cursor: pointer;
+    }
+    label {
+        font-size: 1em;
     }
-  }
\ No newline at end of file
+    .personalize input{
+        font-size: .75em;
+        margin-right: 5%;
+    }
+    button {
+        font-family: 'Quicksand', sans-serif;
+        background-color: #134d71;
+        border-radius: 5px;
+        border: none;
+        color: white;
+        padding: 5%;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 1em;
+        margin: 5% 2%;
+        cursor: pointer;
```

Comment:
Are you changing all of these properties?

Reply:
I believe so. Is there a way I need to shorten this?